### PR TITLE
Update mmd-app-requirements.md

### PR DESCRIPTION
**Pull request by dannoakes**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by jaimeodell** [comment]
I believe you've assigned this to the incorrect person. I am not familiar with this project.

Thanks,
Jaime

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 1]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
 |Management area  |Microsoft Managed Desktop approach  |
 |---------|---------|
 |Device configuration or policy management     |  Microsoft Intune       |
-|Application management     | Microsoft Intune and Company Portal        |
+|Application management     | Applications must be deployed using Microsoft Intune and Company Portal. See Secuirty and Protection applications for for guidance and conditions      |
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:26]:
```suggestion
|Application management     | Applications must be deployed using Microsoft Intune and Company Portal. See "Non-Microsoft security and protection applications" later in this topic for guidance and conditions      |
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 2]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
 |Identity and access management     | See [Identity and access management](security.md#identity-and-access-management)        |
 |Network security     | See [Network security](security.md#network-security)        |
 |Information security     |  See [Information security](security.md#information-security)       |
 |Data recovery     | OneDrive for Business        |
-|Core productivity     | Microsoft 365 Apps for enterprise    |
+|Microsoft 365 Apps for Enterprise     |  Microsoft Managed Desktop maintains Microsoft 365 Apps for Enterprise      |
 |Browser     | Microsoft Edge        |
 
 
+Microsoft Managed Desktop might monitor other software running on managed devices. If it negatively impacts device management, device security, performance, or reliability you may be required to take action.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:37]:
```suggestion
Microsoft Managed Desktop might monitor other software running on managed devices. If it negatively impacts device management, device security, performance, or reliability, you might be required to take action.
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 3]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
 
+# Non-Microsoft Security and Protection Applications
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:39]:
```suggestion
# Non-Microsoft security and protection applications
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 4]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
 
-Microsoft Managed Desktop might monitor other software running on managed devices. If it negatively impacts device management, device security, performance, or reliability, you might be required to request an [exception to the service plan](customizing.md).
+Microsoft Managed Desktop does not test with or ensure continued functioning as part of the service for non-Microsoft security and protection applications. This includes but is not limited to applications commonly referred to as agents, drivers, kernel drivers or non-Microsoft Windows services running as local system. The Microsoft Managed Desktop service does not take dependencies on any external applications or services deployed by IT or Security admins as part of service delivery – see https://docs.microsoft.com/microsoft-365/managed-desktop/sevice-description
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:41]:
```suggestion
Microsoft Managed Desktop does not test with or ensure continued functioning as part of the service for non-Microsoft security and protection applications. This includes but is not limited to applications commonly referred to as agents, drivers, kernel drivers, or non-Microsoft Windows services running as a local system. The Microsoft Managed Desktop service does not take dependencies on any external applications or services deployed by IT or Security admins as part of service delivery. See [Microsoft Managed Desktop service description](/managed-desktop/service-description/?view=o365-worldwide) for more information.
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 5]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+
+These types of apps may provide unrestricted privilege to the operating system and expose Windows to additional threats. See [Technology Used]technologies.md) to identify the capabilities included as part of Microsoft Managed Desktop and the recommended Microsoft 365 native capabilities an IT or Security Admin should implement.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:43]:
```suggestion
These types of apps may provide unrestricted privilege to the operating system and expose Windows to additional threats. See [Technology Used](technologies.md) to identify the capabilities included as part of Microsoft Managed Desktop and the recommended Microsoft 365 native capabilities an IT or Security admin should implement.
```

Comment (JohanFreelancer9) [reply microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:43]: Please also confirm that this link is correct.

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 6]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+
+The requirements listed below must be met if it is not possible for IT or Security Admins to implement or integrate with Microsoft Managed Desktop service options and recommended Microsoft 365 native capabilities. Microsoft Managed Desktop might monitor other software running on managed devices. If it negatively impacts device management, device security, performance, or reliability you may be required to take action.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:45]:
```suggestion
The following requirements must be met if it is not possible for IT or Security admins to implement or integrate with Microsoft Managed Desktop service options and recommended Microsoft 365 native capabilities. Microsoft Managed Desktop might monitor other software running on managed devices. If it negatively impacts device management, device security, performance, or reliability, you might be required to take action.
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 7]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+
+•	Services implemented by Microsoft Managed Desktop must not be impeded or blocked
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:47]:
```suggestion
 - Services implemented by Microsoft Managed Desktop must not be impeded or blocked.
 
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 8]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+•	Settings managed by Microsoft Managed Desktop must not be modified or impeded
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:48]:
```suggestion
 - Settings managed by Microsoft Managed Desktop must not be modified or impeded.
 
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 9]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+•	Microsoft Defender Antivirus state must be Active and therefore must not be modified - see the compatibility matrix to understand if the application being installed will change this state https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-antivirus/microsoft-defender-antivirus-compatibility
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:49]:
```suggestion
 - Microsoft Defender Antivirus state must be _Active_ and therefore must not be modified; see [Microsoft Defender Antivirus compatibility](/windows/security/threat-protection/microsoft-defender-antivirus/microsoft-defender-antivirus-compatibility) to understand if the application being installed will change this state.
 
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 10]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+•	Only Credential Providers included with Windows 10 may be used – see Credential Providers included with Windows 10  https://docs.microsoft.com/en-us/windows/win32/secauthn/credential-providers-in-windows
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:50]:
```suggestion
 - Only Credential Providers included with Windows 10 may be used; see [Credential Providers in Windows 10](/windows/win32/secauthn/credential-providers-in-windows) for details.
 
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 11]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+•	System drive encryption included in the settings documented as managed by Microsoft, must not be modified or impeded
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:51]:
```suggestion
 - System drive encryption included in the settings documented as managed by Microsoft must not be modified or impeded.
 
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 12]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+•	The required network endpoints must not be impeded or blocked – see https://docs.microsoft.com/microsoft-365/managed-desktop/get-ready/network
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:52]:
```suggestion
 - The required network endpoints must not be impeded or blocked; see [Network configuration for Microsoft Managed Desktop](/microsoft-365/managed-desktop/get-ready/network) for details.
 
```

File: microsoft-365/managed-desktop/service-description/mmd-app-requirements.md [hunk 13]
```diff
@@ -23,17 +23,31 @@ Microsoft Managed Desktop requires that we manage devices using a specific appro
...
+•	IT and Security Admins must use the Security operations process in response to threats on endpoints, https://docs.microsoft.com/microsoft-365/managed-desktop/service-description/security-operations
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/mmd-app-requirements.md:53]:
```suggestion
 - IT and Security admins must use the security operations process in response to threats on endpoints; see [Security operations in Microsoft Managed Desktop](/microsoft-365/managed-desktop/service-description/security-operations) for details.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@jaimeodell please review the feedback and update the PR.  Thanks.

**Comment by yogkumgit** [comment]
@dannoakes Please update the PR with suggested changes. Thanks

**Comment by tiburd** [comment]
@dannoakes 

Can you please review this PR and determine whether the proposed changes are still technically accurate?

Please **@ mention** me in your reply. If the changes are still technically accurate, I will propose them in the private repo (and close this PR) so that I can confirm the changes on staging before they go live.

Thank you!

**Comment by dannoakes** [comment]
This can be closed.

**Pull request closed without merging**